### PR TITLE
Document @forward ... with

### LINKED ISSUE #383: Document @forward with
See https://github.com/sass/sass/issues/2744

### PR DESCRIPTION
**Pull request by nex3**
See sass/sass#2744
Closes #383

### CODE REVIEW

File: source/documentation/at-rules/forward.html.md.erb [hunk 1]
```diff
@@ -186,3 +186,56 @@ variables (including the `$`).
...
+  $border-radius: 0.25rem !default;
+  $box-shadow: 0 0.5rem 1rem rgba($black, 0.15) !default;
+
+  code {
+    border-radius: $border-radius;
+    box-shadow: $box-shadow;
+  }
+  ---
+  // _opinionated.scss
+  @use 'library' with (
```
Comment (mirisuzanne) [inline source/documentation/at-rules/forward.html.md.erb]: This should be `@forward` I think?

File: source/documentation/at-rules/forward.html.md.erb [hunk 2]
```diff
@@ -186,3 +186,56 @@ variables (including the `$`).
...
+  $black: #000 !default
+  $border-radius: 0.25rem !default
+  $box-shadow: 0 0.5rem 1rem rgba($black, 0.15) !default
+
+  code
+    border-radius: $border-radius
+    box-shadow: $box-shadow
+  ---
+  // _opinionated.sass
+  @use 'library' with ($black: #222 !default, $border-radius: 0.1rem !default)
```
Comment (mirisuzanne) [inline source/documentation/at-rules/forward.html.md.erb]: `@forward`?

File: source/documentation/at-rules/forward.html.md.erb (hunk 1 above)
Comment (nex3) [reply source/documentation/at-rules/forward.html.md.erb]: Done.

File: source/documentation/at-rules/forward.html.md.erb (hunk 2 above)
Comment (nex3) [reply source/documentation/at-rules/forward.html.md.erb]: Done.

**Pull request merged**